web-api/src/cases/sealInLowerEnvironmentLambda.js

```diff
@@ -0,0 +1,27 @@
+const createApplicationContext = require('../applicationContext');
+
+/**
+ * used for retroactively sealing a case in a lower environment after it is sealed in the Production environment
+ *
+ * @param {object} event the AWS event object received that includes any messages from our SNS subscription
+ * @returns {Promise<*>|undefined} the response to the topic
+ */
+exports.sealInLowerEnvironmentLambda = async event => {
+  const user = { role: 'docketclerk' };
```

Comment:
Could this logic be extracted into an interactor? Also, do we need to wrap this lambda in the genericHandler?

Reply:
Took a stab at moving the logic. lmk if that helps with visibility and testing. Thanks for your feedback!